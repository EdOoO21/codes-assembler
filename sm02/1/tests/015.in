-977 67 507 -926 -677 869 -511 -966 -888 801 648 395 -699 755 422 -248 -510 -761 -308 -45 450 -272 -426 -197 -461 -295
