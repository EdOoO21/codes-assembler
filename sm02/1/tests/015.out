-295
-461
-197
-426
-272
450
-45
-308
-761
-510
-248
422
755
-699
395
648
801
-888
-966
-511
869
-677
-926
507
67
-977
